portal/tests/pageObjects/portal/teach/dashboard_page_new.py

```diff
@@ -35,10 +35,83 @@
 # program; modified versions of the program must be marked as such and not
 # identified as the original program.
 from teach_base_page_new import TeachBasePage
+from selenium.webdriver.support.ui import Select
+
+import time
 
 
 class TeachDashboardPage(TeachBasePage):
     def __init__(self, browser):
         super(TeachDashboardPage, self).__init__(browser)
 
         assert self.on_correct_page('teach_dashboard_page_new')
+
+    def check_organisation_details(self, details):
+        correct = True
+
+        first_field = details.items()[0][0]
```

Comment:
Only if details is a dictionary. Currently the dictionaries are created as they are passed in as arguments.